fiftyone/core/models.py

```
@@ -527,227 +518,100 @@ def _apply_image_model_data_loader(
-
-                        pb.update(len(imgs))
-
-            except Exception as e:
-                if not skip_failures:
-                    raise e
-
-                logger.warning("Sample: %s\nError: %s\n", sample.id, e)
+                    sample.add_labels(
+                        {
+                            int(fn): labels
+                            for fn, labels in zip(fns, labels_frames)
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)